options/options.go

```diff
@@ -76,6 +76,11 @@ func New() (*Options, error) {
 	// Enable scorecard command to use SARIF format.
 	os.Setenv(options.EnvVarEnableSarif, trueStr)
 
+	// Parse relevant env vars.
+	if err := env.Parse(envVars); err != nil {
+		return nil, fmt.Errorf("parsing entrypoint env vars: %w", err)
+	}
```

Comment:
Is there a reason to:
1. create a new struct (`Env`) which overlaps with `Options`
2. parse the env twice

See: https://github.com/ossf/scorecard-action/blob/e55a3ede2c4a2b684ec395a8f5ce054dcb929d61/options/options.go#L82-L84

Unless there's a good reason not to, any env vars that we need that are not already present should be added to `Options`.

Reply:
Originally this was to reduce the number of os calls in optoins.go, but adding them directly to the options struct seems like a better solution.